packages/blockchain/test/pos.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { Block } from '@ethereumjs/block'
+import Common, { Hardfork } from '@ethereumjs/common'
+import { BN } from 'ethereumjs-util'
+import tape from 'tape'
+import Blockchain from '../src'
+import testnet from './testdata/testnet.json'
+
+const buildChain = async (blockchain: Blockchain, common: Common, height: number) => {
+  const blocks: Block[] = []
+  const londonBlockNumber = common.hardforkBlockBN('london')!.toNumber()
+  const genesis = Block.genesis({}, { common })
+  blocks.push(genesis)
+
+  for (let number = 1; number <= height; number++) {
+    let baseFeePerGas: BN = new BN(0)
+    if (number === londonBlockNumber) baseFeePerGas = new BN(1000000000)
```

Comment:
For readability I would prefer if we always used curly braces with if statements (I should enable [eslint/curly](https://eslint.org/docs/rules/curly) hehe), it's just a personal preference but we stick to it pretty well in the monorepo. The only place where it doesn't really bother me is with `return` i.e. `if (statement) return`.